CONTRIBUTING.md

```diff
@@ -113,6 +113,11 @@ following rules before you submit a pull request:
    scale in dimensionality: n_features is expected to be lower than
    100".
 
+-  If your PR includes changes to files under the ``doc/`` or
```

Comment:
I don't think there's a strong reason OS X isn't tested except that we've rarely had problems that behaved differently between OS X and linux. It's not like we're building a GUI or interfacing with other processes.